pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+name = "gen3utils"
+version = "0.4.14"
+description = "Gen3 Library Template"
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/uc-cdis/gen3utils"
+include = [
+  "CHANGELOG.md",
+  "gen3utils/manifest/validation_config.yaml",
+  "LICENSE",
+  "NOTICE",
+  "README.md",
```

Comment:
I think `LICENSE`, `README.md` and the yaml file in source code are included by default.

Reply:
I just built locally and confirmed that those 3 files get included by default.